### PR TITLE
Remove SPA framework

### PR DESCRIPTION
**Pull request by KonyukhovSergey**
https://trello.com/c/ci1FO8VA

### CODE REVIEW

File: styles/framework/common/framework.less [hunk 1]
```diff
@@ -19,11 +19,6 @@ body {
     background: #fff;
 }
 
-.dx-viewport {
```
Comment (KonyukhovSergey) [inline styles/framework/common/framework.less:22]: NOTE:

framework.less = dx.spa.css in artifacts
dx.spa.css = framework CSS styles + reset CSS styles

In this PR, we remove SPA framework CSS styles only. We are going to make a decision on `dx.spa.css` in a separate PR.

**Review by AlekseyMartynov — APPROVED** [review]
The 2nd best PR ever 💪

**Comment by BingoRUS** [comment]
@AlekseyMartynov what is the first one?

**Review by nightskylark — APPROVED** [review]

**Comment by San4es** [comment]
> @AlekseyMartynov what is the first one?

@BingoRUS 
FYI: https://github.com/DevExpress/DevExtreme/pull/7179#pullrequestreview-211147909

File: ts/dx.all.d.ts [hunk 2]
```diff
@@ -1522,273 +1522,6 @@ declare module DevExpress.exporter {
         value?: any;
     }
 }
-declare module DevExpress.framework {
```
Comment (San4es) [inline ts/dx.all.d.ts:1525]: @IlyaKhD Is it ok to do this in the context of this PR?

Comment (IlyaKhD) [reply ts/dx.all.d.ts:1525]: Yes, a PR that updates declarations (doc-comments) is supposed to have corresponding TS updates as well

**Review by IlyaKhD — APPROVED** [review]

File: styles/framework/common/framework.less (hunk 1 above)
Comment (San4es) [reply styles/framework/common/framework.less:22]: FI: https://github.com/DevExpress/DevExtreme/pull/7257

**Pull request merged**